xml/operating_system.xml

```diff
@@ -130,6 +130,15 @@
   <!-- Windows end -->
 
   <!-- Liunx begin -->
+  <fingerprint pattern="^(?i:Alpine Linux\s?(?:v)?(\d+?(?:\.\d+?)*?)?)$">
+    <description>Alpine Linux</description>
+    <example os.vendor="Alpine">Alpine Linux v3.4.0</example>
+    <example os.version="3.4.0">Alpine Linux v3.4.0</example>
```

Comment:
We could collapse this to a single example I think

Reply:
Agreed.  Also, there is no need to test `os.vendor` in this case because it is hardcoded.  If it was capture from a regular expression, like `os.version`, then you should test it.